src/AppInstallerCLITests/YamlManifest.cpp

```diff
@@ -569,7 +572,7 @@ void VerifyV1ManifestContent(const Manifest& manifest, bool isSingleton, Manifes
     auto installer1Dependencies = installer1.Dependencies;
     REQUIRE(installer1Dependencies.HasExactDependency(DependencyType::WindowsFeature, "PreviewIIS"));
     REQUIRE(installer1Dependencies.HasExactDependency(DependencyType::WindowsLibrary, "Preview VC Runtime"));
-    REQUIRE(installer1Dependencies.HasExactDependency(DependencyType::Package, "Microsoft.MsixSdkDepPreview"));
+    REQUIRE(installer1Dependencies.HasExactDependency(DependencyType::Package, "Microsoft.MsixSdkDepPreview", "1.0.0"));

```

Comment:
I was wondering all manifest_good* manifests use the same validation code, and in this change, only 1.5 singleton is modified, and still the tests passed.

Then I found the implementation of HasExactDependency() is incorrect

```
                if (dependency.MinVersion) {
                    if (dependency.MinVersion.value() == minVersion)
                    {
                        return true;
                    }
                }
                else {
                    return true;
                }
```

should be 

```
if (!minVersion.empty())
{
      return minVersion == dependency.MinVersion.value();
}
else
{
     return true;
}                
```

And all v1.1 to v1.5 singleton and multifile manifests should be updated with this